src/services/points.ts

```
@@ -85,6 +85,88 @@ async function fetchInvitesGraph(
     }
 }
 
+type InvitesGraphResponse = {
```

Biome (2.1.2)

[error] Shouldn't redeclare 'InvitesGraphResponse'. Consider to delete it or rename it.

'InvitesGraphResponse' is defined here:


(lint/suspicious/noRedeclare)